src/Api/Checkout.php

```diff
@@ -0,0 +1,152 @@
+<?php
+
+namespace zaporylie\Vipps\Api;
+
+use zaporylie\Vipps\Model\Checkout\Logistics;
+use zaporylie\Vipps\Model\Checkout\MerchantInfo;
+use zaporylie\Vipps\Model\Checkout\PaymentTransaction;
+use zaporylie\Vipps\Model\Checkout\PrefillCustomer;
+use zaporylie\Vipps\Model\Checkout\RequestAmount;
+use zaporylie\Vipps\Model\Checkout\RequestInitiateSession;
+use zaporylie\Vipps\Model\Checkout\ResponseCancelSession;
+use zaporylie\Vipps\Model\Checkout\ResponseGetSessionDetails;
+use zaporylie\Vipps\Model\Checkout\RequestCancelSession;
+use zaporylie\Vipps\Model\Checkout\ResponseInitiateSession;
+use zaporylie\Vipps\Resource\Checkout\CancelSession;
+use zaporylie\Vipps\Resource\Checkout\GetSessionDetails;
+use zaporylie\Vipps\Resource\Checkout\InitiateSession;
+
+/**
+ * Class Checkout.
+ *
+ * @package Vipps\Api
+ */
+class Checkout extends ApiBase implements CheckoutInterface {
+
+    /**
+     * {@inheritdoc}
+     */
+    public function initiateSession(
+      string $client_secret,
```

Comment:
Shouldn't client_secret be passed directly to https://github.com/zaporylie/php-vipps/pull/46/files#diff-6ebb443a052c32288c2745c58ee38f643d1aaff7d34fc1ade90980c8e57d6f36R55? It is not clear to me how what this value is but since it's required for every method then it should be provided one level above, similar to how merchant_serial_number is provided.

Reply:
Done.